pubspec.yaml

```diff
@@ -21,6 +21,7 @@ dependencies:
   cupertino_icons: ^1.0.6
   firebase_analytics: ^10.5.1
   firebase_auth: ^4.10.1
+  flutter_background_service: ^5.0.5
```

Comment:
May you move this below firebase_database to keep the packages in alphabetical order